src/filters/index.js

```diff
@@ -63,20 +63,23 @@ export default {
   /**
    * 12345 => $12,345.00
    *
-   * @param {String} sign
+   * @param {String} currency
+   * @param {Number} places
    */
 
-  currency (value, currency) {
+  currency (value, currency, places) {
```

Comment:
There's not, I never heard about places :sweat_smile:, sorry
@chrisvfritz Places seems fine to me